LayerClient/LayerClient.py

```diff
@@ -237,28 +238,39 @@ def create_conversation(self, participants, distinct=True, metadata=None):
             )
         )
 
-    def update_conversation(self, conversation, metadata=None):
+    def update_conversation(self, conversation_uuid, metadata=None, custom_operations=None):
         '''
         Updates metadata of conversation
 
-        :param conversation: `Conversation` object with `id` not being empty
+        :param conversation_uuid: Conversation uuid. For backwards compatibily accepts also `Conversation` object
         :param metadata: Unstructured data to be passed through to the client.
             This data must be json-serializable.
+        :param custom_operations: Other operations you want to do on the conversation
         :return: `Response` object
         '''
-        return self._raw_request(
-            METHOD_PATCH,
-            self._get_layer_uri(
-                LAYER_URI_CONVERSATIONS,
-                conversation.uuid()
-            ),
-            [
+
+        if not isinstance(conversation_uuid, basestring):
```

Comment:
`basestring` is gone in python 3 - I think `isinstance(uuid, str)` should work on both?